website/docs/providers/notifier_provider/todos/todos.dart

```diff
@@ -0,0 +1,84 @@
+import 'package:flutter/foundation.dart';
+import 'package:riverpod_annotation/riverpod_annotation.dart';
+
+part 'todos.g.dart';
+
+/* SNIPPET START */
+
+// The state of our StateNotifier should be immutable.
+// We could also use packages like Freezed to help with the implementation.
+@immutable
+class Todo {
```

Comment:
Could you use Freezed for this class?

We're in the code-gen variant of the source, so using Freezed makes sense.

Reply:
done